### DISCUSSION TITLE
no screenshots on website? no download

### DISCUSSION THREAD

**Discussion by dataf3l** [pulsar-edit/.github#276] [General]
.

**Comment by savetheclocktower** [comment]
I think this idea could have been expressed more eloquently and diplomatically; but I, too, think it's a bit odd that you don't see any screenshots of the editor on the homepage.

**Comment by dataf3l** [comment]
I profusely apologize for not being more diplomatic, I'm sorry, it was not my intention to be perceived as rude, English isn't my first language

**Reply by savetheclocktower** [reply]
You have nothing to apologize for. I shouldn't have assumed that you were being snide like a Hacker News commenter when there were other possible explanations.

Anyway, we've been working on a blog post about Atom's history, and it wasn't until then that I saw the old home page for [Atom IDE](https://web.archive.org/web/20180315201422/https://ide.atom.io/). I like this screencast and I think something like it would show off the good parts of Pulsar. (I know a lot of the features it demonstrates are specific to Atom IDE, but I think we could do something similar that shows off the built-in features of Pulsar.)

**Comment by Daeraxa** [comment]
You should definitely be seeing a download link front and centre on the homepage:

![image](https://github.com/pulsar-edit/.github/assets/58074586/c608a6a9-41ce-410c-8e6c-87daf519e2d3)

You are right though in that I don't think we have any screenshots on the homepage, we basically want to move away from this entire website framework entirely at some point and work is progressing on that front but we haven't really thought about a homepage revamp in a while.

**Reply by savetheclocktower** [reply]
> You should definitely be seeing a download link front and centre on the homepage:

I think the sentiment of the subject is “I won't download an app if there aren't any screenshots of it on the web site.”

**Comment by LexGear** [comment]
This makes sense. I agree with this.
